packages/circuit-ui/components/Popover/Popover.spec.tsx

```diff
@@ -17,7 +17,7 @@
 
 import { FC } from 'react';
 import { Delete, Add, Download, IconProps } from '@sumup/icons';
-import { Placement } from '@popperjs/core';
+import { Placement } from '@floating-ui/react-dom';
```

Comment:
Really really cool to see all tests pass (except snapshots obvi) after switching the lib!

What you could do is add a new test for the new `offset` prop 🙂 happy to help with that next week